google/cloud/security/common/gcp_api/cloud_resource_manager.py

```diff
@@ -114,16 +114,17 @@ def get_project_iam_policies(self, resource_name, project_numbers):
         """
         projects_stub = self.service.projects()
 
-        try:
-            for project_number in project_numbers:
+        for project_number in project_numbers:
```

Comment:
Got it, I made the parameter generic now.  This makes the api cleaner.  It will just be a bit more work on the caller to handle the output.